spec.html

```diff
@@ -4086,6 +4120,220 @@ <h1>CopyDataBlockBytes ( _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, _count_
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[PrivateFieldValue]]
+            </td>
+            <td>
+              any ECMAScript language value
+            </td>
+            <td>
+              The value of the field.
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-classfielddefinition-record-specification-type">
+      <h1>The ClassFieldDefinition Record Specification Type</h1>
+      <p>The ClassFieldDefinition type is a Record used in the specification of class fields.</p>
+      <p>Values of the ClassFieldDefinition type are Record values whose fields are defined as by <emu-xref href="#table-classfielddefinition-fields"></emu-xref>. Such values are referred to as <dfn>ClassFieldDefinition Record</dfn>s.</p>
+      <emu-table id="table-classfielddefinition-fields" caption="ClassFieldDefinition Record Fields">
+        <table>
+          <tbody>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[Name]]
+            </td>
+            <td>
+              a Private Name, a String value, or a Symbol value.
+            </td>
+            <td>
+              The name of the field.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[Initializer]]
+            </td>
+            <td>
+              an ECMAScript function object, or ~empty~
+            </td>
+            <td>
+              The initializer of the field, if any.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[IsAnonymousFunctionDefinition]]
+            </td>
+            <td>
+              Boolean
+            </td>
+            <td>
+              *true* if there is an Initializer and it is a function definition that does not bind a name, *false* otherwise.
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-private-names">
+      <h1>Private Names</h1>
+      <p>The <dfn>Private Name</dfn> specification type is used to describe a globally unique record which represents a private class element (field, method, or accessor). A Private Name may be installed on any ECMAScript object with PrivateFieldAdd, and then read or written using PrivateFieldGet and PrivateFieldSet.</p>
+
+      <p>Each Private Name holds the following information:</p>
+      <emu-table id="table-private-name-fields" caption="Fields of the Private Name">
+        <table>
+          <thead>
+          <tr>
+            <th>
+              Field
+            </th>
+            <th>
+              Type
+            </th>
+            <th>
+              For which types is it present
```

Comment:
Normally, a column-head is a noun phrase, so: "Types for which it is present".

Moreover, this raises the question "types of what?". In fact, it's referring to the setting of the `[[Kind]]` field, which doesn't define a "type" in the sense that the spec normally means it. So maybe "Values of the [[Kind]] field for which it is present".

Mind you, it troubles me somewhat that the setting of the `[[Kind]]` field has such an effect on the shape of the record.